packages/api/row.go

```diff
@@ -67,10 +70,30 @@ func getRowHandler(w http.ResponseWriter, r *http.Request) {
 
 	q := model.GetDB(nil).Limit(1)
 	table := params["name"]
-	if model.FirstEcosystemTables[table] {
-		q = q.Table("1_"+table).Where("id = ? and ecosystem = ?", params["id"], client.EcosystemID)
+
+	table = converter.ParseTable(table, client.EcosystemID)
+	sc := smart.SmartContract{
```

Comment:
copypaste. Add all that stuff to the CheckAcess function instead of copypasting

Reply:
Fixed